src/main/java/de/muenchen/isi/domain/service/AbfrageService.java

```diff
@@ -61,11 +62,11 @@ public InfrastrukturabfrageModel getInfrastrukturabfrageById(final UUID id) thro
      *
      * @param abfrage zum Speichern
      * @return das gespeicherte {@link InfrastrukturabfrageModel}
-     * @throws UniqueViolationException falls der Name der Abfrage {@link InfrastrukturabfrageModel#getAbfrage().getNameAbfrage} ()} bereits vorhanden ist
+     * @throws UniqueViolationException   falls der Name der Abfrage {@link InfrastrukturabfrageModel#getAbfrage().getNameAbfrage} ()} bereits vorhanden ist
      * @throws OptimisticLockingException falls in der Anwendung bereits eine neuere Version der Entität gespeichert ist
```

Comment:
DataIntegrityViolationException ist nicht in den Javadoc der Methoden beschrieben.